manifest.json

```diff
@@ -0,0 +1,144 @@
+{
+  "dependencies": {
```

Comment:
The idea is intall_third_party.py will depend on manifest.json so stuff in install_third_party.py will be removed.